knowledge-base/grid-filter-operator-dropdown.md

```diff
@@ -0,0 +1,457 @@
+---
+title: Use Filter Operator Drop Down in Filter Template
+description: Learn how to implement and reuse a filter operator selector in a filter row template, as well as a button that clears the current column filter.
+type: how-to
+page_title: How to Use Filter Operator Drop Down in Filter Template
+slug: grid-kb-filter-operator-dropdown
+position: 
+tags: telerik, blazor, grid, treelist, filter
+ticketid: 
+res_type: kb
+---
+
+## Environment
+
+<table>
+    <tbody>
+        <tr>
+            <td>Product</td>
+            <td>Grid for Blazor, <br /> TreeList for Blazor</td>
+        </tr>
+    </tbody>
+</table>
+
+
+## Description
+
+This KB article answers the following questions:
+
+* How to implement a filter operator dropdown in a [Grid filter row template (`FilterCellTemplate`)]({%slug grid-templates-filter%})?
+* How to reuse the filter operator logic and the clear filter button logic across multiple filter cell templates?
+
+
+## Solution
+
+The example below shows how to mimic built-in features, which exist in the `FilterRow` `FilterMode` of the Grid and TreeList:
+
+* Operator DropDownList, which allows users to select fitler operators like "equal to", "starts with", etc.
+* Clear Button, which allows users to clear the column's filter.
+
+The sample also shows a few additional optional features:
+
+* Highlight the Filter Operator DropDownList when the column is filtered. See the `IsFilterActive` parameter of `FilterOperatorList`.
+* Choose the available options in the Filter Operator DropDownList. See the `FilterOperators` parameter of `FilterOperatorList`.
```

Comment:
You might want to link to an appropriate resource, if available.

Reply:
This is a reference to custom code in the example. There is no applicable resource to link.